perception_launch/launch/object_recognition/detection/lidar_based_detection.launch.xml

```diff
@@ -112,8 +111,6 @@
     <include file="$(find-pkg-share detected_object_validation)/launch/object_lanelet_filter.launch.xml">
       <arg name="input/object" value="objects_before_filter"/>
       <arg name="output/object" value="$(var output/objects)"/>
-      <arg name="filtering_range_param" value="$(find-pkg-share perception_launch)/config/object_recognition/detection/object_lanelet_filter.param.yaml" />
```

Comment:
@miursh Is it different as you expected?

Reply:
@yukke42 No, I need to keep this line. I will fix tier4_perception_launch.

Reply:
@.miursh says that this line should not be deleted.

Reply:
changed at [ada8e61](https://github.com/tier4/autoware_launch/pull/408/commits/ada8e61706487ee44d9aec16aff9c409a27c4e46)